src/main/java/gov/usgs/ngwmn/model/MediationType.java

```diff
@@ -5,10 +5,11 @@
 
 
 public enum MediationType {
-	BelowLand,
-	AboveDatum,
-	ASCENDING,
-	DESCENDING;
+	DEFAULT,   // null protection use
+	AboveDatum,// well data use
+	BelowLand, // well data use
+	ASCENDING, // general stats use
+	DESCENDING;// general stats use
```

Comment:
I added a default sort because of the next feature to come, as mentioned, an override will be optional. If the mediation is not been set then we know by default. Null could have been used but that has other issues.